logic/index.ts

```diff
@@ -8,7 +8,8 @@ import { PlatformAddress } from "codechain-sdk/lib/key/classes";
 export async function giveCCC(
     context: Context,
     to: string,
-    amount: string
+    amount: string,
+    contentId: string
```

Comment:
How about postId?
We are already using "post" to point both tweet and Facebook post.

Reply:
It's more intuitive than `contentId`. Fixed.